src/AuthenticationRequest.ts

```diff
@@ -135,10 +136,10 @@ function createURIFromJWT(
   jwt: string
 ): SIOP.AuthenticationRequestURI {
   const schema = 'openid://';
-  const pejs: PEJS = new PEJS();
+  const peAgent: PresentationExchangeAgent = new PresentationExchangeAgent();
   const optionalPD = extractDataFromPath(requestPayload, '$..presentation_definition');
   if (optionalPD && optionalPD.length) {
-    const validationResult: Validated | Validated[] = pejs.validateDefinition(optionalPD[0].value);
+    const validationResult: Validated | Validated[] = peAgent.validatePresentationDefinition(optionalPD[0].value);
     if (validationResult[0].message != 'ok') {
```

Comment:
Given we now have a class we control, can we move the error if the message is not 'ok' not to that method?